roles/analysis/defaults/main.yml

```diff
@@ -1,2 +1,18 @@
 ---
 # defaults file for analysis
+analysis_packages_centos7:
```

Comment:
So, convert2rhel is only one package that contains everything... I think we could make this a bit different and assume that convert2rhel will be installed as part of the `common` role. 

I don't think we need to make a difference between an analysis package for centos7/8/9 and an conversion package for centos7/8/9, we can make a general solution inside the common role and reuse that for both analysis and convert.

Reply:
Removed